File: cclapi/Models/Usuarios.cs
namespace cclapi.Models
{
    public class Usuarios(int Id, string Nombre, string Email, string Contraseña)
    {
        public int Id { get; set; } = Id;
        public required string Nombre { get; set; } = Nombre;
        public required string Email { get; set; } = Email;
        public required string Contraseña { get; set; } = Contraseña;
    }
}

File: mobileapp/inventario/lib/models/usuarios.dart
class Usuarios {
  final int id;
  final String nombre;
  final String email;
  final String contrasena;

  Usuarios(
      {required this.id,
      required this.nombre,
      required this.email,
      required this.contrasena});

  factory Usuarios.fromJson(Map<String, dynamic> json) {
    return Usuarios(
      id: json['Id'],
      nombre: json['Nombre'],
      email: json['Email'],
      contrasena: json['Contraseña'],
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'Id': id,
      'Nombre': nombre,
      'Email': email,
      'Contraseña': contrasena,
    };
  }
}
